src/crs_linter/rules/crs_tag.py

```diff
@@ -0,0 +1,51 @@
+
+from crs_linter.lint_problem import LintProblem
+from crs_linter.rule import Rule
+
+
+class CrsTag(Rule):
```

Comment:
In the meantime, an addition was made to the [original code](https://github.com/coreruleset/crs-linter/blob/0c09eeb43ce761e74256917d4c243fe0ebd58335/src/crs_linter/linter.py#L827-L840): we have added a new function to check whether the rule has a tag for its filename. I think we need this here too. For details please check #27